dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/logging/source/AccessLogSource.java

```diff
@@ -471,13 +578,28 @@ private static SetterFormatter createSetterFormatter(Configuration config, Forma
             formatters[0] = populateDefaultFormatters(CollectorConstants.KEYS_JSON);
         } else if (jsonAccessLogFieldsConfig.equals("logFormat")) {
             formatters[1] = populateCustomFormatters(fieldsToAddJson, CollectorConstants.KEYS_JSON);
+
         }
 
         if (jsonAccessLogFieldsLogstashConfig.equals("default")) {
             formatters[2] = populateDefaultFormatters(CollectorConstants.KEYS_LOGSTASH);
         } else if (jsonAccessLogFieldsLogstashConfig.equals("logFormat")) {
             formatters[3] = populateCustomFormatters(fieldsToAddLogstash, CollectorConstants.KEYS_LOGSTASH);
         }
+
+        if (accessLogFieldsTelemetryConfig.equals("default")) {
+            formatters[4] = populateDefaultTelemetryFormatters(CollectorConstants.KEYS_TELEMETRY_LOGGING);
+
+            //Include the requestFirstLine for Telemetry to be used as the body where applicable
+            fieldSetters.add((ald, alrd) -> ald.setRequestFirstLine(AccessLogFirstLine.getFirstLineAsString(alrd.getResponse(), alrd.getRequest(), null)));
+
+        } else if (accessLogFieldsTelemetryConfig.equals("logFormat")) {
+            formatters[5] = populateCustomTelemetryFormatters(fieldsToAddTelemetryLogging, CollectorConstants.KEYS_TELEMETRY_LOGGING);
+
+            //Include the requestFirstLine for Telemetry to be used as the body where applicable
+            fieldSetters.add((ald, alrd) -> ald.setRequestFirstLine(AccessLogFirstLine.getFirstLineAsString(alrd.getResponse(), alrd.getRequest(), null)));
+        }
+
```

Comment:
Just add the line here once, instead of having duplicate lines in the if/else
```suggestion
fieldSetters.add((ald, alrd) -> ald.setRequestFirstLine(AccessLogFirstLine.getFirstLineAsString(alrd.getResponse(), alrd.getRequest(), null)));
```

Reply:
This would add the setter for all the other formatters as well which we want to avoid.